style.css

```diff
@@ -1,574 +1,491 @@
+  }
 
-.skills > ul {
-  width: 100%;
-  height: 135px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 600;
-  font-size: 18px;
-  line-height: 27px;
-  letter-spacing: -0.0525em;
-  list-style-type: disc;
-  margin-left: 50px;
-}
+  form > input {
+    height: 52px;
+    padding-top: 4px;
+    padding-bottom: 33px;
+    padding-left: 16px;
+    border-radius: 8px;
+  }
 
-.about > button:hover {
-  cursor: pointer;
-  background-color: #fff;
-  color: #1a2236;
-  border: 1px solid #1a2236;
-}
+  #cbn-b {
+    width: 100%;
+    height: 43px;
+    left: 40px;
+    top: 4652px;
+    background: #1a2236;
+    color: #fff;
+    margin-bottom: 40px;
+  }
 
-.btn:hover {
-  cursor: pointer;
-  background-color: #fff;
-  color: #e2505c;
-  border: 1px solid #e2505c;
-}
+  .footer {
+    background: #1a2236;
+    cursor: pointer;
+    width: 100%;
+    height: 150px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    overflow: hidden;
+  }
 
-.form-section {
-  display: flex;
-  flex-direction: column;
-  background-image: url(./images/image1.png);
-  background-repeat: no-repeat;
-}
+  #arrow {
+    float: right;
+    margin-left: 170px;
+    color: #92dcdc;
+    padding-top: 0;
+    align-items: center;
+  }
 
-form {
-  display: flex;
-  flex-direction: column;
-  gap: 15px;
-  width: 24%;
-}
+  #cv {
+    height: 24px;
+    width: 100%;
+    left: 16px;
+    border-radius: nullpx;
+    color: #92dcdc;
+    padding-top: 32px;
+    padding-left: 16px;
+    padding-right: 1px;
+    font-family: "Poppins", sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+  }
 
-form > input {
-  height: 52px;
-  padding-top: 4px;
-  padding-bottom: 33px;
-  padding-left: 16px;
-  border-radius: 8px;
-}
+  .socialicons {
+    display: flex;
+    justify-content: space-around;
+    align-content: center;
+    height: 23px;
+    width: 100%;
+    left: 1px;
+    border-radius: 8px;
+    padding-top: 0;
+    margin-bottom: 30px;
+  }
 
-.form-section > p {
-  width: 25%;
-  height: 64px;
-  left: 16px;
-  margin-top: 1px;
-}
+  #message {
+    border-radius: 8px;
+    padding-left: 16px;
+  }
 
-.form-section > h1 {
-  width: 100%;
-  height: 50px;
-  left: 16px;
-  font-family: "Playfair Display", sans-serif;
-  font-size: 50px;
-  font-weight: 400;
-  line-height: 50px;
-  text-align: left;
-  padding: 1px;
-  margin-top: 106px;
-}
+  .geo-image,
+  #dt-works,
+  #dot-image,
+  .desktopmenu {
+    display: none;
+  }
 
-#cbn-b {
-  width: 100%;
-  height: 43px;
-  left: 40px;
-  top: 4652px;
-  background: #1a2236;
-  color: #fff;
-  margin-bottom: 40px;
-}
+  #menu {
+    height: 100vh;
+    background-color: #1c528a;
+    flex-direction: column;
+    align-items: center;
+    background-image: url(./images/image_geometry_menu_2.svg);
+    background-repeat: no-repeat;
+    background-position: bottom;
+    background-size: contain;
+    display: none;
+  }
```

Comment:
- [ ] Your mobile menu works as expected, but its style is quite different from the Figma template. Kindly make it look like the template, please use the same background color, center the menu options, and make it on top of your page. Currently, when the user opens the mobile menu, the menu opens below the navbar, and it is possible to see the hamburger button, on small screens the 'X' button does not appear, and the user can scroll down and accidentally leave the mobile menu. Kindly make sure all these requirements are met to create a better user experience.
 - Your menu:
 - 
<img width="367" alt="image" src="https://user-images.githubusercontent.com/61919617/210647368-642fd4b2-e33c-4f62-9aa3-bf2d71e43af4.png">

 - Template menu:
 - 
<img width="289" alt="image" src="https://user-images.githubusercontent.com/61919617/210647417-e019e719-1d9c-4803-9d78-35fa00d7a14b.png">

Reply:
@leonardodiasb Thanks for the review..